controllers/storagecluster/cephcluster.go

```diff
@@ -605,11 +608,22 @@ func getNetworkSpec(sc ocsv1.StorageCluster) rookCephv1.NetworkSpec {
 	return networkSpec
 }
 
-func newExternalCephCluster(sc *ocsv1.StorageCluster, monitoringIP, monitoringPort string) *rookCephv1.CephCluster {
+func newExternalCephCluster(sc *ocsv1.StorageCluster, monitoringIP, monitoringPort string) (*rookCephv1.CephCluster, error) {
 	labels := map[string]string{
 		"app": sc.Name,
 	}
 
+	maxLogSize, err := resource.ParseQuantity("500Mi")
```

Comment:
I think we should use must parse, As this is set in the controller itself and can not be overridden by the user. Same for the internal mode as well.

Reply:
Okay, make sense